playbooks/roles/deploy-osh/tasks/helm-run.yml

```diff
@@ -1,12 +1,14 @@
 ---
 # Inspired by https://github.com/openstack/openstack-helm-infra/blob/master/roles/build-helm-packages/tasks/setup-helm-serve.yaml
 - name: Copy systemd unit into place for helm server if it is not present
+  become: yes
```

Comment:
is this really needed? We are already using the root user for deployment so it should not be necessary to do this unless Im missing something?

Reply:
I am not sure, I was also surprised.
I was just syncing the code with current master and those changes were introduced with https://github.com/SUSE-Cloud/socok8s/pull/273 without specific explanation

Reply:
yikes

Reply:
when we run the ansible playbooks on deployer, we should be able to run not as a root user, which is a security recommendation/practice we've been following.